packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```diff
@@ -14,7 +14,7 @@
 
 Summary: Universal command-line interface for Foreman
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.20
+Version: 2.0
```

Comment:
Should we make it `2.0.0`? Otherwise it might bump to `2.1` and the question becomes whether that's for Foreman 2.1.0 or Foreman 2.0.1. This should also be reflected in hammer itself.

Reply:
I kept this inline with previous nightlies (e.g. 0.20). In the stable branch we actually do have e.g. 0.19.3, so once we branch this will become 2.0.0 indeed.

Reply:
I'd advise to make 2.0.0 everywhere to make it clear we follow a x.y.z scheme.

Reply:
I'm not sure the current pipelines will properly pick it up but i can try.